### ISSUE TITLE
Update the main logo

### ISSUE THREAD

**Issue by inattee** [thunderbird/thunderblog#61]
Please update the currently logo to a new blog-specific one attached
![Blog_Logo](https://github.com/thunderbird/thunderblog/assets/16977891/64421b5a-a159-4f07-8dd2-1c0db654049a)
.

**Comment by devmount** [comment]
Thank you Natalie! I think the best solution would be if you could set the logo for yourself in the WP customizer. I'll investigate this.

**Comment by inattee** [comment]
Thanks Andreas - I tried to do it through WP but it would appear those functionalities are disabled.

**Comment by devmount** [comment]
Yes, this has to be implemented on theme level before you can use this. I'll give an update soon.

**Comment by devmount** [comment]
Aaaand done. Once #62 is merged and released, you can set the logo yourself under Customizer > Site identity > Logo.

**Comment by inattee** [comment]
Amazing, thank you!
